test/Error/outofmem.js

```diff
@@ -3,18 +3,7 @@
 // Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
 //-------------------------------------------------------------------------------------------------------
 
-try
-{
-    var longString = "A";
-    for (var i = 0; i < 31; i++)
-        longString += longString;
-    WScript.Echo(longString.toString());
-}
-catch (e)
-{
-    WScript.Echo(e.name);
-    WScript.Echo(e.message);
-    WScript.Echo(e.description);
-    WScript.Echo(e.number);
-}
-
+var longString = "A";
+for (var i = 0; i < 31; i++)
+   longString += longString;
+WScript.Echo(longString.toString());
```

Comment:
@dilijev Wondering how we could test the changes in `printException` without forcing an uncaught exception situation ? This is also what I've understood from @ianwjhalliday 's comment